charts/base-cluster/templates/monitoring/kube-prometheus-stack/_grafana-config.yaml

```diff
@@ -13,6 +13,12 @@ securityContext:
     type: RuntimeDefault
 containerSecurityContext: {{- include "base-cluster.prometheus-stack.containerSecurityContext" (dict) | nindent 2 }}
 resources: {{- include "common.resources" .Values.monitoring.grafana | nindent 2 }}
+{{- if .Values.monitoring.grafana.persistence.enabled }}
+persistence:
+  enabled: true
+  size: {{ .Values.monitoring.grafana.persistence.size | default "10Gi" }}
+  storageClassName: {{ .Values.monitoring.grafana.persistence.storageClassName | default "default" }}
```

Comment:
See https://github.com/teutonet/teutonet-helm-charts/blob/main/charts/base-cluster/templates/monitoring/loki/loki.yaml#L36

Reply:
please take a look at the latest changes

Reply:
Still valid